app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/showscreen/ShowScreen.kt

```diff
@@ -0,0 +1,671 @@
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            IconButton(
+                onClick = navigateBack
+            ) {
+                Icon(
+                    imageVector = Icons.AutoMirrored.Default.ArrowBack,
+                    contentDescription = null,
+                    tint = MaterialTheme.colorScheme.onPrimary,
+                    modifier = Modifier.size(dimensionResource(R.dimen.show_screen_app_bar_navigation_icon_size))
+                )
+            }
+            Spacer(
+                modifier = Modifier.width(dimensionResource(R.dimen.show_screen_app_bar_spacer_height_1))
+            )
+            Text(
+                text = "Go back",
+                style = MaterialTheme.typography.titleSmall.copy(
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 16.sp
+                )
+            )
+        }
+            IconButton(
+                onClick = addToFavorites,
+                modifier = Modifier
+                    .background(
+                    shape = CircleShape,
+                    color = MaterialTheme.colorScheme.onTertiary
+                )
+                    .size(dimensionResource(R.dimen.show_screen_app_bar_icon_button_size))
+            ) {
+                Icon(
+                    imageVector = Icons.Outlined.FavoriteBorder,
+                    tint = MaterialTheme.colorScheme.onPrimary,
+                    contentDescription = null,
+                    modifier = Modifier.size(dimensionResource(R.dimen.show_screen_app_bar_favorite_icon_size))
+                )
+            }
+    }
+}
+
+@Composable
+private fun ShowScreenCoverImage(
+    imagePath: String,
+    modifier: Modifier = Modifier
+){
+     val brush =
+        Brush.verticalGradient(
+            colorStops =
+            arrayOf(
+                0.0f to MaterialTheme.colorScheme.onBackground,
+                0.95f to MaterialTheme.colorScheme.surfaceContainer,
+            ),
+        )
+
+    Box(
+        modifier = modifier
+    ){
+        AsyncImage(
+            model = imagePath,
+            modifier = Modifier.fillMaxSize(),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            placeholder = painterResource(R.drawable.placeholder_image)
+        )
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(brush = brush)
+        )
+    }
+}
+
+@Composable
+private fun ShowScreenMediaBasicInfo(
+    title:String, date:String,
+    language:String, progress: Int,
+    duration: String?, seasons:Int?,
+    modifier: Modifier = Modifier
+) {
+
+    Row(
+        modifier = modifier,
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.Center
+    ) {
+        Box(
```

Comment:
I would probably move this to `Rating` composable

Reply:
done